core/templates/dev/head/components/search_results_directive.html

```diff
@@ -0,0 +1,27 @@
+{% block base_url %}
+<base href="/search/">
```

Comment:
Actually -- this whole thing is in the wrong place. It shouldn't be in a directive, since directives are small components that can be embedded anywhere. It should instead by in gallery.html (which is turning into search.html, right?).